minidump/src/minidump.rs

```diff
@@ -1426,9 +1417,22 @@ impl MinidumpModuleList {
 
     /// Return a `MinidumpModule` whose address range covers `address`.
     pub fn module_at_address(&self, address: u64) -> Option<&MinidumpModule> {
-        self.modules_by_addr
-            .get(address)
-            .map(|&index| &self.modules[index])
+        let nearest_module = match self
+            .sorted_modules
+            .binary_search_by_key(&address, |i| self.modules[*i].base_address())
```

Comment:
wait do we really get weird overlapping modules? gross that this can't use rangemap :(